src/models/__tests__/civilUnion.test.js

```diff
@@ -227,6 +227,245 @@ describe('The civilUnion model', () => {
     })
   })
 
+  describe('if the Address field is international', () => {
+    it('AlternateAddress is required', () => {
+      const testData = {
+        Address: {
+          street: '123 Test St',
+          city: 'London',
+          country: { value: 'United Kingdom' },
+        },
+      }
+
+      const expectedErrors = ['AlternateAddress.required']
+
+      expect(validateModel(testData, civilUnion))
+        .toEqual(expect.arrayContaining(expectedErrors))
+    })
+
+    describe('if HasDifferentAddress is "Yes"', () => {
+      it('AlternateAddress must be a PO address', () => {
```

Comment:
This is a valid check, but this is impossible to actually make the UI do, right?

Reply:
yup!